templates/templates.go

```diff
@@ -55,29 +112,27 @@ func getEnvArray(envName string) ([]string, error) {
 	return strings.Split(env, ","), nil
 }
 
-func renderTemplate(templateText string) (templateResultBuffer bytes.Buffer, err error) {
-
-	// Create a FuncMap with which to register the function.
-	funcMap := template.FuncMap{
-		"getEnv": func(key string) (string, error) {
-			return getEnv(key)
-		},
-		"getEnvArray": func(key string) ([]string, error) {
-			return getEnvArray(key)
-		},
+func required(env interface{}) (string, error) {
```

Comment:
do we need to use an `interface` here :thinking:

Reply:
Kind of, if we want to have a custom error handling to inform user that value is missing. Otherwise user will be prompted with this message (it's not that bad):
`template: templateCli:2:12: executing "templateCli" at <.ENV2>: invalid value; expected string`

I changed type to string.